events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -25,7 +25,7 @@
         TEN_SECONDS_MS = 10 * ONE_SECOND_MS,
         ZONE_COLOR_INDEX = 19,
         HALF_MULTIPLIER = 0.5,
-        FIRST_WAIT_TO_COUNT_AVATARS = 1500,
+        FIRST_WAIT_TO_COUNT_AVATARS = 1000,
```

Comment:
why this change?

Reply:
Whoops, it's arbitrary, I tried messing with it to understand the senquencing of events at the end of the countdown.  1000 or 1500 doesn't actually make a difference in testing it out.

Reply:
OK. what if the delay were 0? would that still work?

Reply:
It does work, but the incorrect sound effect cuts off the end of the countdown music so it's nicer to have a lil buffer.